priv/repo/data/courses.json

```diff
@@ -1270,5 +1270,17 @@
     "language_names": ["Gleam"],
     "framework_names": [],
     "tool_names": []
+  },
+  {
+    "name": "Programming for Complete Beginners",
+    "youtube_course_id": "PLBLdRr-v59vwnKvmvLtcgmAnsb2K1Ta_M",
+    "type": "playlist",
+    "locale": "en",
+    "image_url": "https://i.ytimg.com/vi/syfSoca5_xg/hqdefault.jpg",
+    "published_at": "2024-03-18T18:36:15Z",
+    "youtube_channel_id": "UCNJfpwGuf3D1jt5WHVx1qLw",
+    "language_names": [],
```

Comment:
During the course, there is not lang being used? If that is the case I think is OK to have those as empty

I think the only downside is that they will not match when searching by select box.

Reply:
There is Python. It will be used frequently to demonstrate concepts in future videos. But Python itself will not be the main focus of the series.

I did think about putting in Python, but decided not to when I read this [comment from Daniel](https://github.com/danielbergholz/techschool.dev/pull/36#issuecomment-1984947322). I'd prefer if it is included though. What do you suggest?

Reply:
I have another suggestion. We can have "Other" as an option for each of the category, ensuring courses beyond web development can be added. What do you think?

Reply:
This is an excellent question. For now, I think it's better to leave all of these empty. But in the future, I have plans to add a new filter called `fundamentals`, where we are going to add courses like this one, introduction to programming, networking basics, intro to computer science etc